### PR TITLE
fix userspace integration in trinityx

### PR DESCRIPTION
**Pull request by cedriccastagnede**

### CODE REVIEW

**Comment by cedriccastagnede** [comment]
fix userspace integration in trinityx

**Comment by dchirikov** [comment]
We need to wait until fix for centos-release-7.5 rpm is landed

**Comment by dchirikov** [comment]
Looks good

File: site/roles/trinity/environment-modules/tasks/main.yml [hunk 1]
```diff
@@ -19,10 +19,7 @@
     path: '/usr/share/Modules/init/.modulespath'
     line: '{{ modulefiles_path }}/{{ item }}'
   with_items:
-    - '/modulegroups'
-    - '/cv-standard'
-    - '/site-local'
-    - '' 
+    - '/groups'
```
Comment (kostty) [inline site/roles/trinity/environment-modules/tasks/main.yml]: should be `groups` without a slash?

**Comment by kostty** [comment]
The comment block needs to be updated to match the new directory structure:
```
# Contents of the shared modulefiles directory:
# /modulefiles_path/
#   |-- cv-advanced       advanced CV modules, not available by default
#   ¦-- cv-standard       standard CV modules, available by default
#   |-- site-local        site-local modules, available by default
#    -- modulegroups      modulefiles to load groups (advanced, local, etc)
```

**Comment by dchirikov** [comment]
`controller.yml` should be changed too. Now it's defining `modulefiles_path` as `/trinity/shared/modulefiles/groups`, but not `/trinity/shared/modules/groups`

**Comment by dchirikov** [comment]
same for `trinity-image.yml`

**Comment by dchirikov** [comment]
Fixes are in #342

**Pull request closed without merging**